.goreleaser.yaml

```diff
@@ -0,0 +1,30 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
```

Comment:
It's not important, but I'd prefer to remove comments like this for something more descriptive to this specific file.